src/quo2/foundations/shadows.cljs

```diff
@@ -1,66 +1,74 @@
+                                  (update-in [:soft :shadow-offset :height] -)
+                                  (update-in [:medium :shadow-offset :height] -)
+                                  (update-in [:intense :shadow-offset :height] -)
+                                  (update-in [:strong :shadow-offset :height] -))]
+    {:dark  {:normal dark-normal :inverted dark-normal-inverted}
+     :light {:normal light-normal :inverted light-normal-inverted}}))
 
-(defn- get-scales
-  [inverted?]
-  (if (theme/dark?)
-    {:shadow-1 {:shadow-color   (colors/alpha colors/neutral-100 0.5)
-                :shadow-offset  {:width  0
-                                 :height (get-inverted inverted? 4)}
-                :elevation      3
-                :shadow-opacity 1
-                :shadow-radius  20}
-     :shadow-2 {:shadow-color   (colors/alpha colors/neutral-100 0.64)
-                :shadow-offset  {:width  0
-                                 :height (get-inverted inverted? 4)}
-                :elevation      4
-                :shadow-opacity 1
-                :shadow-radius  20}
-     :shadow-3 {:shadow-color   (colors/alpha colors/neutral-100 0.64)
-                :shadow-offset  {:width  0
-                                 :height (get-inverted inverted? 12)}
-                :elevation      8
-                :shadow-opacity 1
-                :shadow-radius  20}
-     :shadow-4 {:shadow-color   (colors/alpha colors/neutral-100 0.72)
-                :shadow-offset  {:width  0
-                                 :height (get-inverted inverted? 16)}
-                :shadow-opacity 1
-                :shadow-radius  20
-                :elevation      15}}
-    {:shadow-1 {:shadow-color   (colors/alpha colors/neutral-100 0.04)
-                :shadow-offset  {:width  0
-                                 :height (get-inverted inverted? 4)}
-                :elevation      1
-                :shadow-opacity 1
-                :shadow-radius  16}
-     :shadow-2 {:shadow-color   (colors/alpha colors/neutral-100 0.08)
-                :shadow-offset  {:width  0
-                                 :height (get-inverted inverted? 4)}
-                :elevation      2
-                :shadow-opacity 1
-                :shadow-radius  16}
-     :shadow-3 {:shadow-color   (colors/alpha colors/neutral-100 0.12)
-                :shadow-offset  {:width  0
-                                 :height (get-inverted inverted? 12)}
-                :elevation      5
-                :shadow-opacity 1
-                :shadow-radius  16}
-     :shadow-4 {:shadow-color   (colors/alpha colors/neutral-100 0.16)
-                :shadow-offset  {:width  0
-                                 :height (get-inverted inverted? 16)}
-                :shadow-opacity 1
-                :shadow-radius  16
-                :elevation      13}}))
-
-(def normal-scale (get-scales false))
-
-(def inverted-scale (get-scales true))
+(defn get
```

Comment:
Looks better, I was thinking about a function called `get-shadows` instead of just `get`, because of shadowing the `get` fn in the `shadows` namespace, but since we are giving meaninful `:as` names while requiring, it would be worse (I prefer `shadows/get` more than `shadows/get-shadows`) .

I mean, I agree to this change unless a better name than `get` is found

Reply:
I think it's quite idiomatic to use `get` or any other core function when using `:refer-clojure :exclude`. Since we're doing this, we're not shadowing the var really because it was removed from the env.

I'd be happy with other names, but I gotta say @ulisesmac, `shadows/get` is neat since that's all there is to this namespace. I almost went for `shadows/get-shadows`, but felt the repeated name was meh. In the codebase we have plenty of uses of `:refer-clojure` + `:exclude` so I guess it's all good 🤷🏼